pyanaconda/ui/cockpit/anaconda-webui/Makefile.am

```diff
@@ -57,6 +58,13 @@ bots:
 	if [ -n "$$COCKPIT_BOTS_REF" ]; then git -C bots fetch --quiet --depth=1 origin "$$COCKPIT_BOTS_REF"; git -C bots checkout --quiet FETCH_HEAD; fi
 	@echo "checked out bots/ ref $$(git -C bots rev-parse HEAD)"
 
+# FIXME: we need to download this from somewhere if not existing
+$(ISO_FILE):
+	test ! -e test/boot.iso && curl https://kojipkgs.fedoraproject.org/compose/rawhide/latest-Fedora-Rawhide/compose/Server/x86_64/os/images/boot.iso --output test/boot.iso
```

Comment:
As you already define `$(ISO_FILE)`, please consistently use the variable. You also don't need the `test`, that's what make already does for you.

Reply:
I suggest to use https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Server/x86_64/os/images/boot.iso instead, to make use of the mirror network. It's a more "official" URL than the koji one.